### PR TITLE
Add Aria Labels to Carousel Arrows

### PR DESCRIPTION
**Pull request by NRohrer**
PR adds aria labels to the left and right arrows of the Carousel component to help with keyboard / screen reader accessibility needs.

### CODE REVIEW

File: packages/react-storefront/src/Carousel.js [hunk 1]
```diff
@@ -222,6 +222,7 @@ export default class Carousel extends Component {
             <div className={classes.arrows}>
               {selectedIndex !== 0 && (
                 <IconButton
+                  aria-label='previous'
```
Comment (bedrosenator) [inline packages/react-storefront/src/Carousel.js]: Double quotes for html/jsx attrs " are preferable than single '

Comment (NRohrer) [reply packages/react-storefront/src/Carousel.js]: Updated to use double quotes instead.

**Comment by Siggnja** [comment]
This seems like an awful specific way to hardcode an aria label in the carousel component.
What if I wanted to localize the aria labels?

**Comment by egormesyats** [comment]
I need aria-labels in Carousel too. 
Please, consider more generic way of passing props for prev/next buttons here: https://github.com/moovweb/react-storefront/pull/391
Thanks!

**Pull request closed without merging**